src/graphql/node/crud.rs

```diff
@@ -82,6 +81,29 @@ impl NodeMutation {
                 .parse::<u32>()
                 .map_err(|_| "invalid customer ID")?;
 
+            let agents: Vec<review_database::Agent> = agents
+                .into_iter()
+                .map(|new_agent| {
+                    let draft = match new_agent.draft {
+                        Some(draft) => Some(
+                            draft
+                                .try_into()
+                                .map_err(|_| Error::new("Failed to convert agent draft"))?,
```

Comment:
@sophie-cluml, @BLYKIM,

This only informs users that a failure occurred, without specifying which agents the conversion failed on. I think it would be necessary to address that in a separate issue later. Could you create an issue regarding it?

Reply:
https://github.com/aicers/review-web/issues/387 관련 이슈 생성해두었습니다.